tests/Avalonia.Markup.Xaml.UnitTests/Xaml/BindingTests_RelativeSource.cs

```diff
@@ -78,6 +78,87 @@ public void Binding_To_First_Ancestor_Works()
         }
 
         [Fact]
+        public void Binding_To_First_Ancestor_Without_AncestorType_Uses_LogicalTree()
```

Comment:
Hmm, I still think that this implicit switching of trees depending on whether `AncestorType` is specified or not could be confusing. I'd prefer to require `AncestorType` when `Tree=Visual` (following WPF) and require `Tree=Logical` to switch over to the logical tree where `AncestorType` doesn't need to be specified. What do you think?

Reply:
I'll make `AncestorType` be required when `Tree=Visual`. Sounds reasonable to me.

The automatic switch isn't happening any more. I just forgot to remove the test.